pipeline/dbt/models/staging/sources/imilo/_imilo__models.yml

```
@@ -0,0 +1,110 @@
+version: 2
+
+models:
+  - name: stg_imilo__offres
+    columns:
+      - name: id
+        data_tests:
+          - unique
+          - not_null
+          - dbt_utils.not_empty_string
+      - name: structure_id
+        data_tests:
+          - not_null
+          - relationships:
+              to: ref('stg_imilo__structures')
+              field: id
+      - name: nom
+        data_tests:
+          - not_null
+          - dbt_utils.not_empty_string
+      - name: date_maj
+        data_tests:
+          - not_null
+          - dbt_utils.accepted_range:
+              min_value: "now() - interval '2 years'"
+              config:
+                severity: warn
+      - name: date_creation
+        data_tests:
+          - not_null:
+              config:
+                severity: warn
+          - dbt_utils.not_constant
+      - name: thematiques
```

pour tous les champs qui sont supposés réutilisés nos référentiels, on peut vérifier que c'est le cas.

e.g.

```suggestion
      - name: thematiques
        data_tests:
           - relationships:
               to: ref('thematiques')
               field: value
```